src/loaders/browser-agent.js

```diff
@@ -0,0 +1,27 @@
+import { Agent } from "./agent";
```

Comment:
This represents the "prebuilt" "one-agent" type loader we could expose to NPM.  All features included, on/off but no bundle size change.